llvm/test/CodeGen/NVPTX/cluster-dim.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_90 | FileCheck %s
+; RUN: %if ptxas-12.0 %{ llc < %s -march=nvptx64 -mcpu=sm_90 | %ptxas-verify -arch=sm_90 %}
```

Comment:
I'd also add a run with pre-sm90 GPU, and switch to auto-generated checks, so it captures the "no-attributes here" case for older GPUs.

Reply:
Sounds good. Updated.